### PR TITLE
ein bug fix

### PR DESCRIPTION
**Pull request by joeyszhao**
Changed type from number to text to fix the ein bug

### CODE REVIEW

**Comment by crchong1** [comment]
@joeyszhao can you actually make the front end automatically standardize all EINs in this format: 00-0000000 when they actually input the form? this could take the form of automatically inserting the dash at the 3rd position when they type stuff in, and then disallowing all other positions to have a dash. This will help provide front end clarity.

File: src/components/SignUp/OrganizationInformation.tsx [hunk 1]
```diff
@@ -7,7 +7,9 @@ import {
   isValidPhoneNumber, isValidUSState, isValidZipCode, isValidOrgName,
   isValidEIN,
 } from '../../lib/Validations/Validations';
-import CompleteSignupFlow from './CompleteSignupFlow';
```
Comment (crchong1) [inline src/components/SignUp/OrganizationInformation.tsx:10]: jw why are we removing this?

File: src/components/SignUp/OrganizationInformation.tsx [hunk 2]
```diff
@@ -7,7 +7,9 @@ import {
...
+// import CompleteSignupFlow from './CompleteSignupFlow';
+import 'jquery-mask-plugin';
+import $ from "jquery";
```
Comment (crchong1) [inline src/components/SignUp/OrganizationInformation.tsx:12]: jquery is an okay temporary fix, but lets try to find another way to do it. This is bc react is supposed to be a replacement of jquery. I would even be fine if you wrote a simple function that detects the dash and adds it in.

File: src/components/SignUp/OrganizationInformation.tsx [hunk 3]
```diff
@@ -61,6 +63,24 @@ class OrganizationInformation extends Component<Props, State, {}> {
     };
   }
 
+  ConvertNumber(number, format): string {
```
Comment (crchong1) [inline src/components/SignUp/OrganizationInformation.tsx:66]: jw where are we using this function? also can we have some tests around this?

**Review by crchong1 — CHANGES_REQUESTED** [review]

File: src/components/SignUp/OrganizationInformation.tsx (hunk 1 above)
Comment (joeyszhao) [reply src/components/SignUp/OrganizationInformation.tsx:10]: it wasn't used, but i can keep it in if that's what we want?

File: src/components/SignUp/OrganizationInformation.tsx (hunk 3 above)
Comment (joeyszhao) [reply src/components/SignUp/OrganizationInformation.tsx:66]: this was the first method i tried, but i will delete this!

File: src/components/SignUp/OrganizationInformation.tsx (hunk 1 above)
Comment (crchong1) [reply src/components/SignUp/OrganizationInformation.tsx:10]: ah i see- sounds good

**Review by crchong1 — APPROVED** [review]

**Pull request merged**